src/main/java/se/sundsvall/messaging/integration/db/mapper/HistoryMapper.java

```diff
@@ -0,0 +1,75 @@
+package se.sundsvall.messaging.integration.db.mapper;
+
+import com.google.gson.JsonParser;
+import se.sundsvall.messaging.integration.db.entity.HistoryEntity;
+import se.sundsvall.messaging.integration.db.projection.StatsEntry;
+import se.sundsvall.messaging.model.History;
+import se.sundsvall.messaging.model.Message;
+
+import java.time.LocalDateTime;
+
+public class HistoryMapper {
+	private static final String DEPARTMENT = "department";
+
+	private HistoryMapper() {}
+
+	public static History mapToHistory(final HistoryEntity historyEntity) {
+		if (null == historyEntity) {
+			return null;
+		}
+
+		return History.builder()
+			.withBatchId(historyEntity.getBatchId())
+			.withMessageId(historyEntity.getMessageId())
+			.withDeliveryId(historyEntity.getDeliveryId())
+			.withMessageType(historyEntity.getMessageType())
+			.withOriginalMessageType(historyEntity.getOriginalMessageType())
+			.withStatus(historyEntity.getStatus())
+			.withContent(historyEntity.getContent())
+			.withCreatedAt(historyEntity.getCreatedAt())
+			.build();
+	}
```

Comment:
You could use Optionals here instead of explicitly null checking. The functionality would  be the same, but then we don't have to test a null input. This applies to all mapping methods. ::) 

```suggestion
	public static History mapToHistory(final HistoryEntity historyEntity) {
		return Optional.ofNullable(historyEntity).map(history -> History.builder()
		        .withBatchId(historyEntity.getBatchId())
			.withMessageId(historyEntity.getMessageId())
			.withDeliveryId(historyEntity.getDeliveryId())
			.withMessageType(historyEntity.getMessageType())
			.withOriginalMessageType(historyEntity.getOriginalMessageType())
			.withStatus(historyEntity.getStatus())
			.withContent(historyEntity.getContent())
			.withCreatedAt(historyEntity.getCreatedAt())
			.build()).orElse(null);
	}
```